routes/volunteering_events.js

```diff
@@ -1,7 +1,28 @@
 const Router = require('koa-router');
+const VolunteeringEvent = require('../models/VolunteeringEvent');
+const DBMethods = require('./baseDBMethods');
+const VolunteeringOfferRequestBaseModel = require('../models/VolunteeringOfferRequestBase');
 
 const router = new Router();
 
-// Add sub-routes here using router.get(...), router.post(...), etc.
+function getOfferRequestOfEvent(fieldName) {
+  return async function (ctx) {
+    try {
+      const requestedField = await VolunteeringEvent.findById(ctx.params.id)
```

Comment:
What happens here if the given ID does not exist? Does i return null and is handled by `handleDocResponse`?

Reply:
It will return null, if handleDocResponse handle nulls, it will handle it

Reply:
OK